fastchat/serve/api_provider.py

```diff
@@ -1264,3 +1277,65 @@ def metagen_api_stream_iter(
             "text": f"**API REQUEST ERROR** Reason: Unknown.",
             "error_code": 1,
         }
+
+
+def bailing_api_stream_iter(
+    model_name,
+    messages,
+    temperature,
+    top_p,
+    max_tokens,
+    api_base=None,
+    api_key=None,
+    generation_args=None,
+):
+    url = api_base if api_base else "https://bailingchat.alipay.com/chat/arena"
```

Comment:
replace default url with https://bailingchat.alipay.com/chat/completion ?

Reply:
fix it